tests/json/generator.py

```diff
@@ -8,19 +8,26 @@
 
 class JsonProvider(PythonProvider):
 
-    SUPPORTED_JSON_TYPES = ['str', 'str', 'str', 'str', 'float', 'float',
-                            'int', 'int', 'iso8601', 'uri', 'email']
+    __SUPPORTED_JSON_TYPES = {'str', 'float', 'int', 'iso8601', 'uri', 'email', 'bool'}
```

Comment:
Technically, both of these attributes are protected (one `_`). Is there a reason one should be private and the other one public?